crypto/property/property.c

```diff
@@ -62,6 +63,13 @@ typedef struct {
     LHASH_OF(QUERY) *cache;
 } ALGORITHM;
 
+struct ossl_frozen_method_store_st {
+    /* Property query associated with frozen state */
+    char *propq;
+
+    /* TODO: FREEZE */
+};
+
```

Comment:
The custom `struct ossl_frozen_method_store_st` complicates the implementation in multiple places, and I've managed to achieve the same without it.